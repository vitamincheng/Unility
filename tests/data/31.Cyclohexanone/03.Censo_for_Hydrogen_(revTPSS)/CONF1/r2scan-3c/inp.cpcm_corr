Corrected dielectric energy   =     -0.008572881
Total C-PCM charge            =      0.000000000
C-PCM corrected charges:
    -0.000000001
     0.000251161
     0.000015216
     0.000015255
    -0.000000034
    -0.000000023
    -0.000149250
    -0.000697662
     0.000866103
     0.000866180
    -0.000289316
    -0.000289276
     0.000892179
     0.000892113
     0.000040535
     0.000040670
     0.001043128
     0.001043161
    -0.000005071
    -0.000005072
     0.000030605
     0.000030475
    -0.000000274
    -0.000000274
     0.000431373
    -0.000001157
     0.000431146
    -0.000001159
     0.000509683
     0.000509636
    -0.000000010
    -0.000068803
    -0.000000010
    -0.000068764
    -0.000000349
    -0.000000351
    -0.000000019
    -0.000000019
     0.000979577
     0.000979634
    -0.000050429
    -0.000050426
    -0.000114253
    -0.000249072
    -0.000114178
    -0.000249180
    -0.000002962
    -0.000002965
     0.000819676
     0.000819828
    -0.000007965
    -0.000007966
     0.001284551
     0.001284611
     0.000544482
    -0.000126674
     0.000544431
    -0.000126733
    -0.000000005
    -0.000000005
    -0.000114931
    -0.000114930
    -0.000110414
    -0.000110416
     0.000225989
    -0.000212634
    -0.000029280
     0.001112056
    -0.000001396
    -0.000001393
    -0.000000193
    -0.000000193
    -0.000024639
    -0.000024652
    -0.000000232
    -0.000000231
    -0.000084425
    -0.000237673
    -0.000084343
    -0.000237674
     0.001388457
     0.001388464
    -0.000204601
    -0.000204599
     0.000811211
     0.000811281
    -0.000113391
    -0.000113393
    -0.000001565
    -0.000405839
    -0.000001563
    -0.000406230
    -0.000019655
    -0.000019645
     0.000260623
     0.000260939
    -0.000000022
    -0.000027655
    -0.000101893
    -0.000098087
     0.000634434
    -0.000207165
     0.000909167
    -0.000465592
    -0.000001567
    -0.000000167
    -0.000000000
    -0.000842998
     0.001317022
     0.000459968
    -0.000001180
    -0.000196081
    -0.000048210
    -0.000022098
    -0.001306763
     0.001092308
    -0.000305205
    -0.000155024
    -0.000001137
    -0.000541973
     0.001337075
     0.000335465
    -0.000014311
    -0.000000099
    -0.000036432
    -0.000000140
    -0.000011960
    -0.000000013
     0.000702706
    -0.000162491
     0.000767813
    -0.000103137
     0.000777595
    -0.000204920
    -0.000113600
    -0.000297938
     0.000468987
    -0.000152793
    -0.000258888
    -0.000000107
     0.000412285
     0.000725599
    -0.000000009
     0.000932934
    -0.000009091
    -0.000032560
    -0.000164017
    -0.000000034
     0.000337467
    -0.000000005
    -0.000067673
     0.000279257
    -0.000000261
    -0.000232379
     0.000394182
     0.000041590
    -0.000000012
     0.000407099
     0.000037294
    -0.000004882
    -0.000041123
     0.000148953
    -0.000000090
    -0.000000304
    -0.000000020
    -0.000002092
    -0.000041793
     0.000731645
    -0.000040066
     0.001287611
     0.001083443
    -0.000002881
    -0.000220573
    -0.000282110
    -0.000169604
    -0.000145359
    -0.000049336
    -0.000000034
     0.000163341
    -0.000015369
    -0.000030773
    -0.000001808
     0.000931926
    -0.000004780
    -0.000050262
    -0.000225434
    -0.000008851
    -0.000115067
    -0.000016799
    -0.000130490
    -0.000008592
     0.000561918
    -0.000002139
    -0.000066829
    -0.000000747
    -0.000175904
    -0.000117054
    -0.000303571
    -0.000073440
    -0.000058132
    -0.000485267
    -0.000234847
    -0.000000136
    -0.000000106
    -0.000124607
     0.000231043
     0.000955761
    -0.000069938
    -0.000291447
    -0.000156424
    -0.000721915
    -0.000106388
    -0.000503182
     0.000608732
    -0.000000383
    -0.000008160
    -0.000989694
    -0.000000013
    -0.000224555
     0.000769688
    -0.000000004
     0.000145021
    -0.000000442
    -0.000091268
    -0.000000003
    -0.000009701
     0.000001753
    -0.000000003
     0.000663725
    -0.000235398
     0.000000265
     0.000137433
    -0.000261244
    -0.000444763
    -0.000300590
    -0.000108450
    -0.000079874
    -0.000000062
    -0.000000097
     0.000631568
     0.000464504
    -0.000003226
    -0.000013589
     0.000552617
    -0.000000015
    -0.000751904
    -0.000000015
     0.000053019
    -0.000062041
    -0.000007981
     0.000408589
    -0.000229479
     0.000000035
     0.000483450
    -0.000000354
    -0.000142183
    -0.000334929
    -0.000008376
     0.000000454
    -0.000141242
    -0.000003133
    -0.000001440
    -0.000000005
    -0.000156028
     0.000712646
    -0.000183302
    -0.000007585
     0.000771447
     0.001017589
    -0.000000730
    -0.000023757
    -0.000127506
    -0.000392438
    -0.000697286
    -0.000023644
    -0.000131801
    -0.000000126
    -0.000029927
    -0.000004918
     0.000186025
    -0.000001449
    -0.000249893
    -0.000000037
    -0.000022255
    -0.000827341
    -0.000005245
    -0.000417430
    -0.000002131
    -0.000008485
    -0.000009868
    -0.000000001
    -0.000471740
    -0.000524798
    -0.000610850
    -0.000419860
    -0.000692007
    -0.000877443
    -0.000932246
    -0.000890595
    -0.000953722
    -0.000006604
    -0.000056377
    -0.000031890
    -0.000078511
    -0.000000991
    -0.000352651
    -0.000001284
    -0.000332441
    -0.000003248
    -0.000291228
    -0.000018891
    -0.000291967
    -0.000608557
    -0.000860145
    -0.000580165
    -0.000889981
    -0.000606860
    -0.000633681
    -0.000632514
    -0.000757150
    -0.000787141
    -0.000912910
    -0.000799074
    -0.000980092
    -0.000271131
    -0.000456929
    -0.000115984
    -0.000474672
    -0.000205432
    -0.000640252
    -0.000169400
    -0.000599290
    -0.000000950
    -0.000000015
    -0.000003183
    -0.000004041
    -0.000000001
    -0.000006799
    -0.000875286
    -0.001064334
    -0.000325604
    -0.000697850
    -0.000908551
    -0.000970466
    -0.000497620
    -0.000412976
    -0.000234167
    -0.000385019
    -0.000072511
    -0.000552699
    -0.000000493
    -0.001082815
    -0.000424156
    -0.000000007
    -0.000036829
    -0.000000048
    -0.000451636
    -0.000298131
    -0.000783540
    -0.000573336
    -0.000924746
    -0.000709270
    -0.000179026
    -0.000507000
    -0.000329615
    -0.000618044
    -0.000976345
    -0.000038243
    -0.001170085
    -0.000270057
    -0.000000057
    -0.000185033
    -0.000345768
    -0.000784729
    -0.000000000
    -0.000370789
    -0.000138854
    -0.001020609
    -0.000756555
    -0.000134969
    -0.000000000
    -0.000003550
    -0.000808281
    -0.000346712
    -0.000803215
    -0.000394654
    -0.001156963
    -0.000681054
    -0.000000042
    -0.000614473
    -0.000053226
    -0.000951707
    -0.000000000
    -0.001121592
    -0.000539659
    -0.000000158
    -0.000932070
    -0.000000001
    -0.000000101
    -0.000466294
    -0.000000008
    -0.000179727
    -0.000618971
    -0.000013341
    -0.000694605
    -0.000545393
    -0.000987263
     0.000000000
    -0.000386700
    -0.000010980
    -0.000010990
    -0.000084669
    -0.000008630
     0.000000018
     0.000000908
     0.000000908
     0.000000069
     0.000000069
    -0.000000002
    -0.000000002
    -0.000004861
    -0.000004856
    -0.001015234
    -0.001015329
    -0.001170247
    -0.001170245
    -0.000025124
    -0.000025105
    -0.000466309
    -0.000466321
    -0.000000548
    -0.000000548
    -0.000059783
    -0.000059783
    -0.001126631
    -0.000948284
    -0.001126921
    -0.000948298
    -0.001038581
    -0.001727322
    -0.000993004
     0.000001281
     0.000001281
    -0.000116777
    -0.000116592
    -0.000000010
    -0.000000010
    -0.000474610
     0.000000010
    -0.000474688
     0.000000010
    -0.000014023
    -0.000014005
    -0.001692473
    -0.001692159
    -0.000000086
    -0.000120620
    -0.000000086
    -0.000120620
     0.000000000
     0.000000000
    -0.000411629
    -0.000448685
    -0.000203209
    -0.000423232
    -0.001408019
    -0.000000016
    -0.000804098
    -0.000000324
    -0.000176190
    -0.000419051
    -0.000747724
    -0.000000001
    -0.000835639
    -0.001090400
    -0.000691410
    -0.000924639
     0.000011423
    -0.000967649
    -0.000071164
    -0.000862482
    -0.000153394
    -0.001470334
    -0.000006803
    -0.001241302
     0.000019597
    -0.000000001
    -0.001192335
    -0.000000176
    -0.000403049
    -0.000980979
    -0.001292835
    -0.000073894
    -0.000590674
    -0.000003511
    -0.000446661
    -0.001149847
    -0.000182462
    -0.000838202
    -0.001407286
    -0.000277879
    -0.000993730
    -0.001313995
    -0.000000002
    -0.000924469
    -0.000685980
    -0.001349664
    -0.001262307
    -0.000000040
    -0.000929443
    -0.000000100
    -0.001103395
    -0.000456929
    -0.000973442
    -0.000006439
    -0.001188558
    -0.000695795
    -0.000000002
    -0.000000002
    -0.000612090
    -0.000813667
    -0.001266352
    -0.001065126
    -0.000882334
    -0.001195425
    -0.001165773
    -0.001243493
    -0.001199007
    -0.000047822
    -0.000000291
    -0.000225085
    -0.000094062
    -0.000347637
    -0.000001292
    -0.000476899
    -0.000002339
    -0.000000000
    -0.000297307
    -0.000000229
    -0.000610934
    -0.000050901
    -0.001081331
    -0.001099856
    -0.001196276
    -0.001023126
    -0.000956571
    -0.001071526
    -0.001171064
    -0.001077714
    -0.001176571
    -0.001158685
    -0.001329078
    -0.001196207
    -0.000741755
    -0.000537827
    -0.000875055
    -0.000591640
    -0.000793370
    -0.000465788
    -0.000888449
    -0.000430797
    -0.000000387
    -0.000012833
    -0.000000113
    -0.000001244
    -0.000019046
    -0.000000002
    -0.001362628
    -0.001252157
    -0.000903359
    -0.000724456
    -0.001231434
    -0.001356114
    -0.001047453
    -0.000874007
    -0.000175895
    -0.000117269
    -0.000303895
    -0.000059433
    -0.000073274
    -0.000485414
    -0.000235170
    -0.000000136
    -0.000000105
     0.000230714
     0.000955897
    -0.000124520
    -0.000069869
    -0.000291443
    -0.000156286
    -0.000503229
     0.000608409
    -0.000721374
    -0.000106294
    -0.000000384
    -0.000008119
    -0.000225347
     0.000768738
    -0.000989026
    -0.000000013
     0.000144773
    -0.000000004
    -0.000000003
    -0.000009643
    -0.000000446
    -0.000091329
     0.000001803
    -0.000000003
     0.000663718
    -0.000235011
     0.000000265
     0.000137747
    -0.000260688
    -0.000108328
    -0.000081106
    -0.000445295
    -0.000300315
    -0.000000062
     0.000631417
     0.000465769
    -0.000000097
    -0.000003217
    -0.000013557
     0.000553088
    -0.000000015
     0.000053355
    -0.000000016
    -0.000751563
    -0.000061821
    -0.000007998
     0.000408871
    -0.000229600
     0.000000035
     0.000483472
    -0.000000356
    -0.000142146
    -0.000334886
    -0.000008351
     0.000000453
    -0.000002886
    -0.000141467
    -0.000000005
    -0.000001449
    -0.000155973
     0.000712998
    -0.000183440
    -0.000007562
     0.000771165
     0.001017625
    -0.000023820
    -0.000128074
    -0.000000733
    -0.000023539
    -0.000131374
    -0.000393342
    -0.000697882
    -0.000004899
     0.000186018
    -0.000000125
    -0.000029890
    -0.000001451
    -0.000250182
    -0.000000070
    -0.000005257
    -0.000417545
    -0.000022367
    -0.000827277
    -0.000002133
    -0.000008661
    -0.000009918
    -0.000000022
    -0.000027703
    -0.000101990
     0.000633435
    -0.000097851
    -0.000207344
     0.000909094
    -0.000465620
    -0.000001571
    -0.000000167
     0.001317207
     0.000459722
    -0.000000000
    -0.000842910
    -0.000001176
    -0.000196042
    -0.000048113
     0.001092243
    -0.000305301
    -0.000022109
    -0.001306732
    -0.000155244
     0.001336694
     0.000334879
    -0.000001134
    -0.000541441
    -0.000014274
    -0.000000100
    -0.000011917
    -0.000000013
    -0.000036444
    -0.000000141
     0.000702708
    -0.000162584
     0.000767775
    -0.000103158
     0.000777718
    -0.000204862
     0.000467957
    -0.000152710
    -0.000113522
    -0.000298210
    -0.000258886
     0.000413144
     0.000725493
    -0.000000107
    -0.000000009
     0.000933215
    -0.000009061
    -0.000032516
     0.000337916
    -0.000000005
    -0.000163903
    -0.000000034
    -0.000067772
     0.000279478
    -0.000000261
    -0.000232354
     0.000394328
     0.000041682
    -0.000000012
     0.000407002
     0.000037429
    -0.000004871
    -0.000040893
     0.000149007
    -0.000000090
    -0.000000020
    -0.000000306
    -0.000041516
    -0.000002097
     0.000731930
    -0.000040053
     0.001287767
     0.001083362
    -0.000002876
    -0.000220600
    -0.000145177
    -0.000049226
    -0.000282168
    -0.000169871
     0.000163043
    -0.000015406
    -0.000000034
     0.000931963
    -0.000004760
    -0.000030793
    -0.000001799
    -0.000050277
    -0.000225457
    -0.000008852
    -0.000130833
    -0.000008601
    -0.000115166
    -0.000016850
     0.000562161
    -0.000002146
    -0.000066970
    -0.000000747
    -0.000411435
    -0.000448624
    -0.000203666
    -0.000423438
    -0.001408035
    -0.000000015
    -0.000803861
    -0.000419231
    -0.000747779
    -0.000000326
    -0.000176648
    -0.000000001
    -0.000835560
    -0.001090315
    -0.000691275
    -0.000924472
    -0.000967631
     0.000011409
    -0.000862397
    -0.000070970
    -0.000153430
    -0.001470248
    -0.000006792
    -0.001241103
    -0.000000001
    -0.001192406
     0.000019708
    -0.000402549
    -0.000000176
    -0.000980953
    -0.001292876
    -0.000073719
    -0.000590426
    -0.000446801
    -0.001149937
    -0.000003526
    -0.000182097
    -0.000838175
    -0.001407242
    -0.000277571
    -0.000993486
    -0.001313994
    -0.000924267
    -0.000000002
    -0.000685901
    -0.001349516
    -0.001262184
    -0.000000041
    -0.000929505
    -0.000000099
    -0.001102615
    -0.000456495
    -0.000973462
    -0.000006410
    -0.001188537
    -0.000695582
    -0.000000002
    -0.000000002
    -0.000612106
    -0.001065267
    -0.000882501
    -0.000813515
    -0.001266279
    -0.001243551
    -0.001199109
    -0.001195386
    -0.001165775
    -0.000225479
    -0.000094296
    -0.000047562
    -0.000000288
    -0.000476615
    -0.000002351
    -0.000346972
    -0.000001296
    -0.000000000
    -0.000611198
    -0.000051074
    -0.000296510
    -0.000000229
    -0.001196295
    -0.001023247
    -0.001081214
    -0.001099867
    -0.001171299
    -0.001078000
    -0.000956384
    -0.001071446
    -0.001329184
    -0.001196426
    -0.001176431
    -0.001158650
    -0.000875174
    -0.000592425
    -0.000741156
    -0.000537307
    -0.000888543
    -0.000431279
    -0.000793235
    -0.000465659
    -0.000012868
    -0.000000114
    -0.000000383
    -0.000019042
    -0.000000002
    -0.000001235
    -0.001362603
    -0.001252256
    -0.000903396
    -0.000724664
    -0.001356303
    -0.001231317
    -0.000874188
    -0.001046914
    -0.000000001
    -0.000471722
    -0.000419946
    -0.000692036
    -0.000524681
    -0.000610603
    -0.000890608
    -0.000953703
    -0.000877391
    -0.000932160
    -0.000031958
    -0.000078619
    -0.000006577
    -0.000056219
    -0.000001288
    -0.000332304
    -0.000000993
    -0.000352235
    -0.000018928
    -0.000292069
    -0.000003244
    -0.000290728
    -0.000580195
    -0.000889911
    -0.000608513
    -0.000859969
    -0.000632655
    -0.000757247
    -0.000606706
    -0.000633457
    -0.000799159
    -0.000980095
    -0.000787058
    -0.000912735
    -0.000116296
    -0.000474766
    -0.000270957
    -0.000456607
    -0.000169469
    -0.000599261
    -0.000205381
    -0.000640089
    -0.000000016
    -0.000003190
    -0.000000945
    -0.000000001
    -0.000006795
    -0.000004021
    -0.000875292
    -0.001064241
    -0.000325590
    -0.000697761
    -0.000970528
    -0.000908406
    -0.000413120
    -0.000497366
    -0.000234133
    -0.000384990
    -0.000072623
    -0.000552567
    -0.000000492
    -0.001082799
    -0.000424180
    -0.000000007
    -0.000451577
    -0.000298218
    -0.000036890
    -0.000000048
    -0.000783459
    -0.000573269
    -0.000924699
    -0.000709232
    -0.000506927
    -0.000179003
    -0.000617992
    -0.000329552
    -0.000976236
    -0.000038217
    -0.001170026
    -0.000270080
    -0.000184879
    -0.000000057
    -0.000784715
    -0.000000000
    -0.000345733
    -0.000370669
    -0.000138706
    -0.001020608
    -0.000756528
    -0.000134890
    -0.000000000
    -0.000808292
    -0.000346667
    -0.000003560
    -0.000803093
    -0.000394516
    -0.001156927
    -0.000681027
    -0.000614351
    -0.000000042
    -0.000951671
    -0.000000000
    -0.000053278
    -0.001121512
    -0.000539629
    -0.000931989
    -0.000000001
    -0.000000159
    -0.000000101
    -0.000466243
    -0.000000008
    -0.000179612
    -0.000618921
    -0.000013316
    -0.000694584
    -0.000545264
    -0.000987242
    -0.000000470
    -0.000422886
    -0.000555849
    -0.000358759
    -0.000555880
    -0.000358885
    -0.000864230
    -0.000758015
    -0.000864239
    -0.000758024
    -0.000071876
    -0.000010115
    -0.000071934
    -0.000010137
    -0.000000004
    -0.000000004
    -0.000044468
    -0.000000160
    -0.000044475
    -0.000000160
    -0.000000000
    -0.000000000
    -0.000179698
    -0.000001565
    -0.000179748
    -0.000001568
    -0.000749237
    -0.000314519
    -0.000749249
    -0.000314566
    -0.000665137
    -0.000541476
    -0.000665177
    -0.000541506
    -0.000875895
    -0.000606036
    -0.000875918
    -0.000606062
    -0.000381550
    -0.000208395
    -0.000381590
    -0.000208479
    -0.000310152
    -0.000175088
    -0.000310145
    -0.000175175
    -0.000002824
    -0.000002828
    -0.000007422
    -0.000007424
    -0.000973674
    -0.000633503
    -0.000398454
    -0.000219485
    -0.000849805
    -0.000849832
    -0.000403184
    -0.000403242
    -0.000403146
    -0.000214855
    -0.000831755
    -0.000831741
    -0.000113045
    -0.000113029
    -0.000475509
    -0.000146629
    -0.000475505
    -0.000146626
    -0.000130384
    -0.000130369
    -0.000001449
    -0.000001449
    -0.000892468
    -0.000892465
    -0.000657538
    -0.000657534
    -0.000744311
    -0.000744293
    -0.000454862
    -0.000454834
    -0.001071758
    -0.001071753
    -0.000094409
    -0.000094398
    -0.000528952
    -0.000003709
    -0.000528935
    -0.000003709
    -0.000740741
    -0.000000161
    -0.000740732
    -0.000000161
    -0.000000020
    -0.000000020
    -0.000541240
    -0.000541208
    -0.000036466
    -0.000036461
    -0.001017022
    -0.001017014
    -0.000444882
    -0.000444863
    -0.000380939
    -0.000380907
    -0.000811574
    -0.000811562
    -0.000888095
    -0.000000019
    -0.000000240
    -0.001148265
    -0.000425739
    -0.000392175
    -0.000392167
    -0.000067014
    -0.000067014
     0.000401181
     0.000469769
     0.000469821
     0.000350307
     0.000097924
     0.000505295
     0.001864274
     0.000505304
     0.001864256
     0.001681440
     0.001681481
     0.001210639
     0.001521085
     0.001210624
     0.001521064
     0.001226039
     0.001226065
     0.000314332
     0.002093646
     0.000315469
     0.002093626
     0.001958846
     0.001958886
     0.001636548
     0.001636548
     0.001540657
     0.001540679
    -0.000000000
     0.001978005
    -0.000000000
     0.001977999
     0.001859279
     0.001859316
     0.000077556
     0.001688449
     0.000077582
     0.001688436
     0.001506427
     0.001506457
     0.001405298
     0.001899329
     0.001405222
     0.001899300
     0.001636261
     0.001636308
     0.001116276
     0.001713675
     0.001116258
     0.001713651
     0.001474713
     0.001474749
     0.000525538
     0.002032178
     0.000525792
     0.002032159
     0.001859549
     0.001859594
    -0.000000561
     0.001780522
    -0.000000562
     0.001780515
     0.001644704
     0.001644736
     0.000871309
     0.000911452
     0.000871305
     0.000911447
    -0.000149442
     0.000503603
    -0.000149441
     0.000503606
     0.000695472
     0.001486486
     0.000696678
     0.001486460
    -0.000058328
     0.001407020
    -0.000058370
     0.001407020
     0.001030203
     0.001030205
     0.001011775
     0.001011778
     0.001135201
     0.001274246
     0.001135190
     0.001274232
    -0.000016509
     0.000913761
    -0.000016508
     0.000913773
    -0.000028999
     0.001957275
    -0.000028381
     0.001957254
     0.001884129
     0.001884153
     0.001410865
     0.001410868
     0.001355687
     0.001355698
     0.002050661
     0.002050671
     0.002008287
     0.002008313
    -0.000116661
     0.001591502
    -0.000116658
     0.001591496
     0.001409621
     0.001409632
     0.001723700
     0.001879673
     0.001723649
     0.001879634
    -0.000038667
     0.001427831
    -0.000038661
     0.001427881
     0.001893729
     0.001893730
     0.001353404
     0.001353412
     0.001043265
     0.001193962
    -0.000000572
     0.000884320
     0.001318775
     0.001256342
     0.001184127
     0.001184111
     0.000380412
     0.000380416
     0.001842789
     0.001842747
     0.001106667
     0.001106582
     0.002143558
     0.002143574
     0.001868310
     0.001868328
     0.000824628
     0.001520075
     0.001308171
    -0.000001938
     0.001578397
     0.001428736
     0.001647457
     0.001647421
     0.000775666
     0.000775674
     0.001947035
     0.001946981
     0.001236506
     0.001236535
     0.001728560
     0.002028711
     0.001728466
     0.002028678
     0.001773164
     0.001773218
     0.001446816
     0.001711288
     0.001446783
     0.001711256
    -0.000000536
     0.001341268
    -0.000000535
     0.001341304
    -0.000003550
     0.001626658
    -0.000003550
     0.001626653
     0.001481334
     0.001481354
     0.001849579
     0.001849583
     0.001774608
     0.001774636
     0.000912748
     0.001566568
     0.000912750
     0.001566554
     0.001348578
     0.001348599
     0.002120498
     0.002120495
     0.002032449
     0.002032483
     0.001373577
     0.001471014
     0.001373553
     0.001470987
    -0.000045043
     0.000953869
    -0.000045048
     0.000953886
     0.001910863
     0.002023923
     0.001910881
     0.002023884
    -0.000013386
     0.001809361
    -0.000013390
     0.001809376
     0.001079711
     0.001407988
     0.001079707
     0.001407980
     0.001152788
     0.001152797
     0.002115208
     0.002115208
     0.002071778
     0.002071799
     0.001512706
     0.001512705
     0.001405621
     0.001405630
     0.001641095
     0.001641104
     0.001606740
     0.001606758
