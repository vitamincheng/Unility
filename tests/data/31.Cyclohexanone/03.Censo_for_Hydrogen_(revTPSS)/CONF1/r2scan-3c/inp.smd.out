----------------------------------------------- 
  Element       SASA      Sigma Z        Total  
     Z         Ang**2 cal/Ang**2/mol   kcal/mol 
----------------------------------------------- 
      H        119.792       0.000       0.000
      C         26.510      91.879       2.436
      O         25.184       2.693       0.068

 Subtotal:     171.485                   2.504
 -----------------------------------------------

  Elements    COT*SASA   Sigma Z,Z'     Total   
    Z,Z'       Ang**2  cal/Ang**2/mol kcal/mol  
 ---------------------------------------------- 
  H   C         80.409     -52.587      -4.229
  H   O          0.000     -28.036      -0.000
  C   C         32.537     -89.718      -2.919
  O   C         15.593      -8.302      -0.129

 Subtotal:                              -7.277
 ---------------------------------------------- 

 NONAQUEOUS SOLVENT CONTRIBUTION (kcal/mol):
 (proportional to SASA, atomic number independent)
 ---------------------------------------------- 
 Subtotal:                               1.660
 ---------------------------------------------- 

 CDS TOTAL (kcal/mol):                  -3.114
...............................................................................
